auth/services/oauth/oauth.go

```diff
@@ -245,7 +245,10 @@ func (s *service) validateRequester(context *validationContext) error {
 	actualName := context.contractVerificationResult.ContractAttribute(contract.LegalEntityAttr)
 	actualCity := context.contractVerificationResult.ContractAttribute(contract.LegalEntityCityAttr)
 	if actualName != context.requesterName || actualCity != context.requesterCity {
-		log.Logger().Warnf("Token request validation failed, legal entity mismatch (expected name=%s, actual name=%s, expected city=%s, actual city=%s)", context.requesterName, actualName, context.requesterCity, actualCity)
+		log.Logger().
+			WithField("organizationName", context.requesterName).
+			WithField("organizationCity", context.requesterCity).
+			Warnf("Token request validation failed, organization name/city does not match (given name=%s, given city=%s)", actualName, actualCity)
```

Comment:
This one now has a combination of information in the fields and the message. Perhaps now all include in the fields. Also the terms `expected` and `actual` were more clear then the new names.

Reply:
Will revert this to using no fields, since they're not used anywhere else.